### ISSUE TITLE
password not detected

### ISSUE THREAD

**Issue by sh1va73** [thelinuxchoice/instainsane#54]
i run the program few times but I have multiple issues:
1. I run the program with a password list in which I included my password, but the program does not detect the password.
2. Also when the list is finished the program does not end, it just hangs
3. when i do CTRL + C i get this message: Passwords not tested due IP BLocking:

**Comment by sh1va73** [comment]
Ok i just tried again and it found the correct password. I dont know what happened before but now it worked.

**Comment by fifitoto** [comment]
Yeah, sometime it works and sometime it doesn't work at all. Actually, most of the time it fails to crack the password. It's just 2 / 10 at success. Must need to fix this glitch or bug or error.
